src/Files.App/Extensions/ComboBoxExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using CommunityToolkit.WinUI.UI;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+
+namespace Files.App.Extensions
+{
+	/// <summary>
+	/// Provides extension for ComboBox.
+	/// </summary>
+	/// <remarks>
+	/// - IsKeepWidthEnabled: Prevents from opening popup at wrong position.
```

Comment:
I would say this comment is not entirely correct. There are definitely cases where one wouldn't need this extension. The comment should probably indicate that in fact this extension is setting min width

Reply:
We use this for ComboBoxes in Settings, which are aligned Right position. When there's no MinWidth popup opens at ugly position, swapping to the right.

Reply:
What you describe is what bug this fixes, the symptom you are adressing. The extension however only sets the min width and the comment should explain exactly that in my opinion.